src/ImGui/Desktop/Fusee.ImGui.Desktop/Templates/ImGuiFilePicker.cs

```diff
@@ -461,33 +440,27 @@ public virtual unsafe void Draw(ref bool filePickerOpen)
             return;
         }
 
-        private List<string> GetFileSystemEntries(string fullName)
+        private List<FileSystemInfo> GetFileSystemEntries(string fullName)
         {
             try
```

Comment:
Why do we need `try` here? Also the exception is never handled.

Reply:
While iterating through all files and folders recursivly we may stumble upon locked files and/or inaccessible folders. When this happens an expection is being thrown. We are neither interested in these folders nor in the fact that something went wrong.